verification/src/transaction_verifier.rs

```diff
@@ -25,6 +37,7 @@ impl<'a> TransactionVerifier<'a> {
             script: ScriptVerifier::new(rtx),
             capacity: CapacityVerifier::new(rtx),
             inputs: InputVerifier::new(rtx),
+            valid_since: ValidSinceVerifier::new(&rtx.transaction, &block_context),
```

Comment:
Call `self.valid_since.verify()?;` in `verify`.

https://github.com/nervosnetwork/ckb/pull/372/files#diff-d0ea0a76c2e66e05fbcf6f957b6eccf4R44